Paganism/Interpreter/Data/Extensions/Extension.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Paganism.PParser.AST.Enums;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Paganism.PParser;
+using Paganism.PParser.AST;
+using Paganism.PParser.Values;
+using Paganism.Exceptions;
+using System.Collections.ObjectModel;
+
+#nullable enable
+namespace Paganism.Interpreter.Data.Extensions
+{
+    internal class Extension
```

Comment:
Why internal

Reply:
because extensions can be only internally managed, with APIs you'll be able to add things